talktorials/1_ChEMBL/T1_ChEMBL.ipynb

```diff
@@ -10,7 +10,7 @@
     "\n",
```

Comment:
<p>Do you want to return <code>None</code> here? Either raise an exception or return the unaltered value, I'd say?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Later we use that function in:</p><p> </p><p><code>bioactivities_df.apply(lambda x: convert_to_nM(x['units'], x['value']), axis=1)</code></p><p> </p><p>The unaltered value would be added wrongly to the DataFrame and raising an exception will break the apply function. My workaround therefore was using <code>None</code>. What would you do here?</p>

Reply:
Fair enough! No problems then.